OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -67,7 +67,7 @@ public DecisionService(Bucketer bucketer, IErrorHandler errorHandler, ProjectCon
         /// <param name = "userId" > The userId of the user.
         /// <param name = "filteredAttributes" > The user's attributes. This should be filtered to just attributes in the Datafile.</param>
         /// <returns>The Variation the user is allocated into.</returns>
-        public Variation GetVariation(Experiment experiment, string userId, UserAttributes filteredAttributes)
+        public virtual Variation GetVariation(Experiment experiment, string userId, UserAttributes filteredAttributes)
```

Comment:
why virtual? It shouldn't be virtual.

Reply:
GetVariation is used in mocking so it needs to be virtual.